### DISCUSSION TITLE
Which U.S. President coined the term "Founding Fathers"?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#302] [Galaxy Trivia] [answered]
**Hint:** 29th

**Comment by peterbaccaro** [comment, marked as answer]
Senator Warren G. Harding in his keynote speech at the Republican National Convention of 1916.

**Reply by ParanoidUser** [reply]
Well done! 👏👏 Congratulations on your well deserved [Galaxy Brain](https://github.com/peterbaccaro?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 8 answers), and don't hesitate to post your own questions to the community. 😊🎉
